WWDC 2016 ScreenSaver/WWDC2016ScreenSaver.swift

```diff
@@ -197,23 +197,20 @@ class WWDC2016ScreenSaverView: ScreenSaverView {
     }
     
     func tick() {
-        if self.isAnimating == false {
-            return
-        }
         
-        if textLayers.count == 0 {
-            return
+        func restart() {
```

Comment:
I found it to be lagging in the screensaver for some reason, so I took it out to test, but forgot to put it back in, would probably be better.